### PR TITLE
Fixed announcement title, added some mods, set the feature to true

### PR DESCRIPTION
**Pull request by j10a1n15**

### CODE REVIEW

**Comment by j10a1n15** [comment]
Merge today pls

File: data/main_menu.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
         "discord_invite_code" : "v4PU3WeH7z"
     },
 
-    "prank_sound": false,
+    "prank_sound": true,
```
Comment (Su386yt) [inline data/main_menu.json:11]: 🤨

Comment (j10a1n15) [reply data/main_menu.json:11]: Huh why not, tomorrow is the day

Comment (j10a1n15) [reply data/main_menu.json:11]: Better early then late, we do have the date check itself

**Pull request merged**